content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,245 @@
+      <td>
+        <p>Specification of which user the HostProcess container should run as is required for the pod spec.</p>
+        <p><strong>Allowed Values</strong></p>
+        <ul>
+          <li><code>NT AUTHORITY\SYSTEM</code></li>
+          <li><code>NT AUTHORITY\Local service</code></li>
+          <li><code>NT AUTHORITY\NetworkService</code></li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards"><tt>runAsNonRoot</tt></a></td>
+      <td>
+        <p>Because HostProcess containers have privileged access to the host, the <tt>runAsNonRoot</tt> field cannot be set to true.</p>
+        <p><strong>Allowed Values</strong></p>
+        <ul>
+          <li>Undefined/Nil</li>
+          <li><code>false</code></li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Manifesto de Exemplo (excerto) {#manifest-example}
+
+```yaml
+spec:
+  securityContext:
+    windowsOptions:
+      hostProcess: true
+      runAsUserName: "NT AUTHORITY\\Local service"
+  hostNetwork: true
+  containers:
+  - name: test
+    image: image1:latest
+    command:
+      - ping
+      - -t
+      - 127.0.0.1
+  nodeSelector:
+    "kubernetes.io/os": windows
+```
+
+## Volume mounts
+
+Contêineres HostProcess suportam a capacidade de montar volumes dentro do espaço 
+de volume do contêiner.
+As aplicações em execução dentro do contêiner podem acessar montagens de volume 
+diretamente por meio de caminhos relativos ou absolutos. Uma variável de ambiente 
+`$CONTAINER_SANDBOX_MOUNT_POINT` é definida na criação de contêineres, e fornece 
+o caminho absoluto do host para o volume do contêiner. Caminhos relativos 
+são baseados na configuração `.spec.containers.volumeMounts.mountPath`.
+
+### Exemplo {#volume-mount-example}
+
+Para acessar os tokens da conta de serviço, as seguintes estruturas de caminho 
+são suportadas no contêiner:
+
+`.\var\run\secrets\kubernetes.io\serviceaccount\`
+
+`$CONTAINER_SANDBOX_MOUNT_POINT\var\run\secrets\kubernetes.io\serviceaccount\`
+
+## Limites de recursos
+
+Limites de recursos (disco, memória, número de cpus) são aplicados ao job todo.
+Por exemplo, com um limite de 10MB definido, a memória alocada para qualquer 
+objeto job HostProcess será limitado em 10MB. 
+Este é o mesmo comportamento que outros tipos de contêiner do Windows.
+Esses limites seriam especificados da mesma maneira que são atualmente, para qualquer 
+orquestrador ou runtime sendo usado. A única diferença está no cálculo do uso 
+de recursos de disco usados para rastreamento de recursos, devido à diferença em 
+como contêineres HostProcess são inicializados.
+
+## Escolha uma conta de usuário
+
+Contêineres HostProcess tem a capacidade de executar como uma das três contas 
+de serviço do Windows suportadas:
+
+- **[LocalSystem](https://docs.microsoft.com/windows/win32/services/localsystem-account)**
```

Comment:
There is the article in pt-br. What do you think about update the link?
https://learn.microsoft.com/pt-br/windows/win32/services/localsystem-account